src/content/general/architecture/capi/index.md

```diff
@@ -0,0 +1,227 @@
+
+##### Operators (Custom Resource plus controller)
+
+The most common pattern to extend Kubernetes is called [Operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/). It allows to define a new [Kubernetes resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) (called Custom Resource) and apply functionality to it. 
+
+In Giant Swarm we leverage the operator pattern to extend the management cluster and provide a great user experience to our customers.
+
+*Organization operator*
+
+This operator is in charge of reconciling `Organization` custom resources. The functionality of the operator is pretty straightforward. It creates and deletes the organization namespace when the given resource is created.
+
+To learn more about organizations please read [the related documentation](https://docs.giantswarm.io/general/organizations/).
+
+*RBAC operator*
+
+We have built an RBAC operator with the goal of maintaining up to date permissions between the different organizations and users on the management cluster so [you can isolate your teams and ensure access level in a granular way](https://docs.giantswarm.io/ui-api/management-api/authorization/).
+
+*DNS operator*
+
+We built an operator to manage the DNS entries for all the endpoints we need to expose to our customers. All our management clusters have a base domain used to allocate the API, OIDC endpoint or UI. This operator listens to cluster resource and provides the right records.
+
+*Deletion Blocker Operator*
+
+Kubernetes ensures resources are deleted in the right order by using `finalizers`, a way to create dependencies between resources. In specific cases, some CRs don't leverage finalizers, which can stop the deletion process when they are deleted too early and leave the cluster in an inconsistent state. Deletion Blocker Operator ensure that specific CRs aren't deleted until specific conditions are met.
+
+*App operator*
+
+The [App Platform](https://docs.giantswarm.io/app-platform/) is a system we built to deliver Cloud Native Apps in a multi cluster fashion. App operator is the main code running in management clusters to manage App custom resources and make sure the applications and configuration are delivered correctly across the different workload clusters.
+
+*Cluster API operators*
+
+In [Cluster API](https://cluster-api.sigs.k8s.io/) there is a set of generic operators that all Cluster API providers run to address the common bootstrap and management actions on a Kubernetes cluster lifecycle which include:
+
+- `Kubeadm Bootstrap`controller which is in charge of providing a cloud init configuration to turn a machine into a Kubernetes node.
```

Comment:
```suggestion
- `Kubeadm Bootstrap` controller which is in charge of providing a cloud init configuration to turn a machine into a Kubernetes node.
```

Reply:
In general, I would question the use of backticks here. We use them to indicate that something is "code", to show that exact spelling is important.

If you just want to emphasise, you can use `**` to make the name bold.